csrc/kernel_cache.h

```diff
@@ -222,7 +222,8 @@ class FusionKernelRuntime {
   using HeuristicsPtr = std::unique_ptr<FusionHeuristics>;
   std::optional<HeuristicsPtr> getMaybeHeuristicsFor(
       const KernelArgumentHolder& args,
-      std::optional<PrimDataType> forced_index_type = std::nullopt);
+      std::optional<PrimDataType> forced_index_type,
+      bool initial_heuristics);
```

Comment:
nitpick, maybe add default for `forced_index_type` as well as `initial_heuristics` so we don't have to go change the benchmark files.